06.bitstream.syntax.md

```diff
@@ -572,6 +574,23 @@ at least one byte of the payload data (including the trailing bit) shall not be
 | }
 {:.syntax }
 
+#### Metadata cropping syntax
+
+| --------------------------------------------------------- | ---------------- |
+| metadata_crop( ) {                                        | **Type**
+|     @@crop_width_minus_1                                  | f(16)
+|     @@crop_height_minus_1                                 | f(16)
+|     @@crop_offset_present                                 | f(1)
```

Comment:
Why don't we make it byte-aligned in the first place? having something like:

```
f(7) reserved
f(1) crop_offset_present;
```
would be better IMO than relying on the `trailing_bits` at the end of the OBU.